plugins/topology/src/components/Topology/TopologyToolbar.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+import {
+  ToolbarItem,
+  Select,
+  SelectOption,
+  SelectVariant,
+  SelectOptionObject,
+} from '@patternfly/react-core';
+import { K8sResourcesContext } from '../../hooks/K8sResourcesContext';
+
+const TopologyToolbar: React.FC<{
+  setClusterContext: React.Dispatch<React.SetStateAction<number>>;
+}> = ({ setClusterContext }) => {
+  const { clusters: k8sClusters } = React.useContext(K8sResourcesContext);
```

Comment:
do we need to take `setSelectedCluster` as props? or

```suggestion
const TopologyToolbar: React.FC = () => {
  const { clusters: k8sClusters , setSelectedCluster} = React.useContext(K8sResourcesContext);
```